microcosm_flask/factories.py

```diff
@@ -16,7 +16,12 @@ def configure_flask(graph):
     app.debug = graph.metadata.debug
     app.testing = graph.metadata.testing
 
-    # TODO: wire in the graph's configuration to Flask's
+    # copy in the graph's configuration for non-nested keys
```

Comment:
Basically: some third-party libraries look in `app.config` for their config. We want to avoid that where possible, but do propagate top-level keys from our configuration. The `BasicAuth` library is a good test case for this feature since it loads its *realm* from the config.